server/rest/src/test/java/org/infinispan/rest/resources/CacheV2ResourceTest.java

```diff
@@ -64,9 +64,9 @@ private ConfigurationBuilder getIndexedPersistedCache() {
       return builder;
    }
 
-   @AfterClass
+   @AfterMethod
```

Comment:
I was going to ask you last night to add a comment on why `@AfterTest` is needed, but GitHub wouldn't allow me to click submit :)